hosted-tenant-base/pom.xml

```diff
@@ -0,0 +1,408 @@
+
+    <!--    <version>7-SNAPSHOT</version>-->
+    <!--    <url>https://github.com/vespa-engine</url>-->
+    <!--    <packaging>pom</packaging>-->
+
+    <groupId>com.yahoo.vespa</groupId>
+    <artifactId>hosted-tenant-base</artifactId>
+    <version>7-SNAPSHOT</version>
+    <name>Base pom for all tenant base poms</name>
+    <description>Parent POM for all Vespa base poms.</description>
+    <url>https://github.com/vespa-engine</url>
+    <packaging>pom</packaging>
+
+    <licenses>
+        <license>
+            <name>The Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
+            <name>Vespa</name>
+            <url>https://github.com/vespa-engine</url>
+        </developer>
+    </developers>
+    <scm>
+        <connection>scm:git:git@github.com:vespa-engine/vespa.git</connection>
+        <developerConnection>scm:git:git@github.com:vespa-engine/vespa.git</developerConnection>
+        <url>git@github.com:vespa-engine/vespa.git</url>
+    </scm>
+
+    <properties>
+        <vespaversion>${project.version}</vespaversion>
+        <test-framework.version>${project.version}</test-framework.version>
+        <target_jdk_version>11</target_jdk_version>
+        <maven-compiler-plugin.version>3.8.0</maven-compiler-plugin.version>
+        <maven-surefire-plugin.version>2.22.0</maven-surefire-plugin.version>
+        <junit.version>5.4.2</junit.version>
+        <endpoint>https://api.vespa-external.aws.oath.cloud:4443</endpoint>
+        <test.categories>!integration</test.categories>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.yahoo.vespa</groupId>
+                <artifactId>container-dependency-versions</artifactId>
+                <version>${vespaversion}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+
+            <dependency>
+                <groupId>org.junit.vintage</groupId>
+                <artifactId>junit-vintage-engine</artifactId>
+                <version>${junit.version}</version>
+            </dependency>
+
+            <dependency>
+                <groupId>org.junit.jupiter</groupId>
+                <artifactId>junit-jupiter-engine</artifactId>
+                <version>${junit.version}</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.yahoo.vespa</groupId>
+            <artifactId>container</artifactId>
+            <version>${vespaversion}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.yahoo.vespa</groupId>
+            <artifactId>container-test</artifactId>
+            <version>${vespaversion}</version>
+            <scope>test</scope>
```

Comment:
We'll have to override the scope to runtime for `cloud-tenant-base`, perhaps we need to move this definition down.

Reply:
can't we just set the scope to runtime in hosted-tenant-base?